BenchmarkSample/benchmark/build.gradle

```diff
@@ -8,6 +8,15 @@ android {
     defaultConfig {
         minSdkVersion 14
         targetSdkVersion 29
+
+        // Change this to enable profiling, can be one of:
+        //   * None
+        //   * Sampled
+        //   * Method
+        //   * ConnectedSampled
```

Comment:
remove ConnectedSampled and ConnectedAllocation - those aren't going to be publicly documented yet, since the UX is so bad right now

Reply:
Done